build.gradle.kts

```diff
@@ -22,16 +22,17 @@ subprojects {
         version.set("0.37.2")
         enableExperimentalRules.set(true)
         verbose.set(true)
+        debug.set(true)
```

Comment:
Should we be leaving debug on?

Reply:
Seems it's not needed, I'll remove this line.